layers/gpuav/spirv/buffer_device_address_pass.h

```diff
@@ -34,17 +35,26 @@ class BufferDeviceAddressPass : public Pass {
     // This is metadata tied to a single instruction gathered during RequiresInstrumentation() to be used later
     struct InstructionMeta {
         const Instruction* target_instruction = nullptr;
+        const Instruction* pointer_inst = nullptr;
         uint32_t alignment_literal = 0;
-        uint32_t type_length = 0;
+        uint32_t access_size = 0;
         bool type_is_struct = false;
     };
 
     bool RequiresInstrumentation(const Function& function, const Instruction& inst, InstructionMeta& meta);
-    uint32_t CreateFunctionCall(BasicBlock& block, InstructionIt* inst_it, const InstructionMeta& meta, bool safe_mode);
+    uint32_t CreateFunctionCall(BasicBlock& block, InstructionIt* inst_it, const InstructionMeta& meta);
 
     // Function IDs to link in
     uint32_t function_range_id_ = 0;
     uint32_t function_align_id_ = 0;
+
+    struct Range {
+        uint32_t begin_instruction = 0;
+        uint32_t begin = vvl::kU32Max;
+        uint32_t end = 0;
```

Comment:
```suggestion
        uint32_t begin = vvl::kU32Max;
        uint32_t end = 0;
```
Given how these are used, they probably should be named something like `min/max_struct_offsets`?

Reply:
sure, `min`/`max` probably better